hack/generator/pkg/codegen/pipeline_status_from_swagger.go

```diff
@@ -106,7 +100,19 @@ type statusTypes struct {
 	resourceTypes resourceLookup
 
 	// otherTypes has all other Status types renamed to avoid clashes with Spec Types
-	otherTypes []astmodel.TypeDefinition
+	otherTypes astmodel.Types
+}
+
+func (st statusTypes) findResourceType(typeName astmodel.TypeName) (astmodel.Type, bool) {
+	if statusDef, ok := st.resourceTypes.tryFind(typeName); ok {
+		klog.V(4).Infof("Swagger information found for %v", typeName)
+		return statusDef, true
+	} else {
+		klog.V(4).Infof("Swagger information missing for %v", typeName)
```

Comment:
Suggest that messages about missing information should be more important than when it's found - maybe make this `V(3)`?

Reply:
We insert an error type (below) so that if the type is used you will get a warning logged. These are diagnostic if you want to see what _would_ be missing for pruned types, so they aren't relevant to normal usage.

Reply:
I was more thinking that having actionable warnings at a more significant level than information means that it's easy to get just those warnings when needed. 

Having both branches at `V(4)` means that you've got no choice other than sift through all the log messages to find the ones where it's missing.

Reply:
I'll bump missing to `V(3)`.